### PR TITLE
Blogs-1303 SMP iStats

### PR DESCRIPTION
**Pull request by mattgreenham**
https://jira.dev.bbc.co.uk/browse/BLOGS-1303

Adding counterName so replicates behaviour of v4.

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: templates/base.html.twig [hunk 1]
```diff
@@ -47,6 +47,10 @@
                     }
                 }
             });
+
+            window.bbcBlogs = {
```
Comment (votemike) [inline templates/base.html.twig]: At what point is this needed? Will this definitely be set by the time it's needed?

Comment (mattgreenham) [reply templates/base.html.twig]: It's needed when the smp is rendered in the body, so I think its fine being set here

**Comment by mattgreenham** [comment]
Test this please

**Comment by mattgreenham** [comment]
Test this please

**Pull request merged**